### PR TITLE
Pull action grammar

### LINKED ISSUE #6628: pull description inside new "no local changes" view
When your current branch is both ahead and behind, this is what the button shows:

I had a peek at the message that was being generated and it looks like the description always shows `commits`:

https://github.com/desktop/desktop/blob/cd783fa43845c4d26d81ef18d38021a59c172ba5/app/src/ui/changes/no-changes.tsx#L478-L483

For reference, for the **Push** action we do pluralize inside the description:

https://github.com/desktop/desktop/blob/cd783fa43845c4d26d81ef18d38021a59c172ba5/app/src/ui/changes/no-changes.tsx#L527-L533

While we're in here the "doesn't" in "that doesn’t exist on your machine" jumped out at me as being grammatically off (if you expand the abbreviation to "that does not exist" it stands out) and I was wondering if we had feels about tweaking it, particularly to avoid using abbreviations in messages like this.

Suggested alternatives:

> The current branch that has commits on GitHub _that do not_ exist on your machine.
> The current branch that has commits on GitHub _which do not_ exist on your machine.

### PR DESCRIPTION
**Pull request by niik**
## Overview

Fix pluralization of 'commits' and implement suggested grammar tweak.

**Closes #6628**

## Release notes
no-notes

### CODE REVIEW

**Comment by niik** [comment]
:apple:

File: app/src/ui/changes/no-changes.tsx [hunk 1]
```diff
@@ -442,11 +442,13 @@ export class NoChanges extends React.Component<INoChangesProps, {}> {
     }
 
     const isGitHub = this.props.repository.gitHubRepository !== null
+    const commitOrCommits = aheadBehind.behind === 1 ? 'commit' : 'commits'
 
     const description = (
       <>
-        The current branch (<Ref>{tip.branch.name}</Ref>) has commits on{' '}
-        {isGitHub ? 'GitHub' : 'the remote'} that doesn’t exist on your machine.
+        The current branch (<Ref>{tip.branch.name}</Ref>) has {commitOrCommits}{' '}
```
[the remark this answers is not in the document]
Comment (shiftkey) [reply app/src/ui/changes/no-changes.tsx]: Paging English native speakers to this thread:

[the remark this answers is not in the document]
Comment (j-f1) [reply app/src/ui/changes/no-changes.tsx]: > The current branch (`branch`) has X commit/commits on GitHub that aren’t on your computer yet.

**Comment by billygriffin** [comment]
@shiftkey Yep, very unfortunate typo on my part 😭

**Comment by shiftkey** [comment]
If someone has feels on refining the words they're welcome to follow up with another PR 📝

**Pull request merged**